recipe/meta.yaml

```diff
@@ -21,17 +22,16 @@ requirements:
     - {{ stdlib('c') }}
   host:
     - python
-    - setuptools >=64.0.0
-    - numpy  # empty version spec will pick up channel pinnings
-    # - numpy >=1.22,<2  # cyrk specific pinnings also enforced
-    - cython >=3.0.0
-    - wheel >=0.38
+    - setuptools  >=64.0.0
+    - numpy  <2.0.0
```

Comment:
Try not pinning here, but let conda-forge choose and appropriate one on a per-Python version basis.